### PR TITLE
Removed the `OrderedDict`s

### PR DESCRIPTION
**Pull request by 1Codealot**
Relates to this issue: https://github.com/H-uru/korman/issues/398

I also have added type annotations :)

### CODE REVIEW

File: korman/nodes/node_responder.py
Comment (dpogue) [inline korman/nodes/node_responder.py:73]: This extra space might be introducing indentation issues. Comments in korman are aligned to the 4-space tab stops.

File: korman/nodes/node_responder.py
Comment (dpogue) [inline korman/nodes/node_responder.py:244]: Unsure if this comment should be removed/reworded, I'll leave it to Hoikas to make that decision

**Comment by 1Codealot** [comment]
Cool, thanks for reviewing!
I'll do those now!

**Comment by 1Codealot** [comment]
What do you think now?

**Review by dpogue — APPROVED** [review]
Looks good to me. Hoikas is much more familiar with the typing stuff, so I'd like his review on that before merging.

Thanks for the pull request and responding to feedback! It's really great to see new contributors 😁

**Comment by 1Codealot** [comment]
> It's really great to see new contributors 😁

You're welcome!

File: korman/nodes/node_responder.py
Comment (Hoikas) [reply korman/nodes/node_responder.py:245]: Yeah, let's leave this in. Korman still has a policy of trying to upgrade in place as much user data as possible.

**Review by Hoikas — COMMENTED** [review]
Oh man, thanks for getting the ball rolling on this. Unfortunately, we're still on a rather old version of Python, so this PR doesn't quite work as-is. I'm seeing errors like:
```
Traceback (most recent call last):
  File "D:\Blender\b4k_20221113_x64\2.79\scripts\modules\addon_utils.py", line 351, in enable
    mod = __import__(module_name)
  File "D:\Blender\b4k_20221113_x64\2.79\scripts\addons\korman\__init__.py", line 20, in <module>
    from . import nodes
  File "D:\Blender\b4k_20221113_x64\2.79\scripts\addons\korman\nodes\__init__.py", line 25, in <module>
    from .node_deprecated import *
  File "D:\Blender\b4k_20221113_x64\2.79\scripts\addons\korman\nodes\node_deprecated.py", line 50, in <module>
    class PlasmaResponderCommandNode(PlasmaDeprecatedNode, bpy.types.Node):
  File "D:\Blender\b4k_20221113_x64\2.79\scripts\addons\korman\nodes\node_deprecated.py", line 58, in PlasmaResponderCommandNode
    "type": "PlasmaRespCommandSocket",
TypeError: 'type' object is not subscriptable
```
From what I can tell, these can be corrected by ensuring that each file you've added the `dict` type annotations to, opt into string-based annotations with `from __future__ import annotations` and be sure to import the `typings` module into the global namespace by `from typing import *`.

Thanks again for the contribution! I look forward to seeing the revisions.

File: korman/nodes/node_responder.py
Comment (Deledrius) [inline korman/nodes/node_responder.py:245]:
```suggestion
        # While this is deprecated I might as well also convert it.
```

**Review by Deledrius — CHANGES_REQUESTED** [review]
Just a minor typo in the comment here.

**Comment by Deledrius** [comment]
Thanks for joining in to make Korman better! :)

**Comment by 1Codealot** [comment]
Thanks for the reviews, I'll change them in a bit!!

**Comment by 1Codealot** [comment]
bruh i did a typo in the commit 💀

**Comment by Deledrius** [comment]
> bruh i did a typo in the commit 💀

No problem, you can amend the commit to edit the commit message. :)

**Review by Deledrius — APPROVED** [review]

**Comment by 1Codealot** [comment]
> No problem, you can amend the commit to edit the commit message. :)

Sorry I can't seem figure out how (I'm doing this on the remote vscode on GH at the moment)

**Comment by Deledrius** [comment]
It looks like it is possible, but a bit convoluted to do in VSCode (https://stackoverflow.com/questions/63882369/can-i-amend-a-commit-made-with-vscode-to-github-repo).  

That's alright.  I, or one of the other maintainers, can fix the commit message.  Don't worry about it. :)

File: korman/nodes/node_avatar.py
Comment (Hoikas) [inline korman/nodes/node_avatar.py:18]: Any time that you do a `from __future__` import, it needs to be the first import statement. This, unfortunately, errors out. You'll need to move it up to the very top of the list of imports.

Comment (1Codealot) [reply korman/nodes/node_avatar.py]: Done!

**Comment by 1Codealot** [comment]
Done the `from __future__ import annotations` movements to the top!

**Comment by 1Codealot** [comment]
Done moving the `from __future__ import annotations` to the top!

**Review by Hoikas — COMMENTED** [review]
Great, this is working without errors now. The only thing I see that still needs to be adjusted is that some of the type hints are referencing the wrong `any`. [`any`](https://docs.python.org/3.7/library/functions.html#any) is a built in function that checks an iterable for any value that is truthy, but I think we need to be using [`Any`](https://docs.python.org/3.7/library/typing.html#typing.Any), which means "any type".

I pushed a very small adjustment to some whitespace on the first commit, btw.

**Comment by 1Codealot** [comment]
> I think we need to be using [Any](https://docs.python.org/3.7/library/typing.html#typing.Any), which means "any type".

Right yep, will do!

**Comment by 1Codealot** [comment]
Done!

**Review by Hoikas — APPROVED** [review]
Excellent! Thanks for helping out. We look forward to more contributions from you in the future! 😄

**Pull request merged**
